tchannel/container/heap.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+
+
+class HeapOperation(object):
+    def cmp(self, i, j):
+        """Compare the items in position i and j of the heap.
+
+        :param i: the first item's position of the heap list
+        :param j: the second item's position of the heap list
+        :return true if items[i] < items[j], otherwise false.
+        """
```

Comment:
The `HeapOperation` can always flip its definition of what `lt` or `cmp` means if it wants a min/max heap. This issue is mostly about naming. If you're calling it `cmp` I would expect it to be the Python `cmp` operation: returns value < 0 if left < right, value = 0 if left = right, and value > 0 if left > right.